src/LineBot/Actions/CameraAction.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Line
+{
+	/// <summary>
+	/// Encapsulates a template camera action.
```

Comment:
The word template should be removed, I forgot to remove it from the other actions...

Reply:
Sure thing! I'll get that straightened out and add the unit tests 👍

Reply:
Feel free to take a peek at the other unit tests for some ideas on how I set those up.